app/packages/operators/src/state.ts

```diff
@@ -670,7 +671,11 @@ export const operatorChoiceState = atom({
 export const recentlyUsedOperatorsState = atom({
   key: "recentlyUsedOperators",
   default: [],
-  effects: [fos.getBrowserStorageEffectForKey("operators-recently-used")],
+  effects: [
+    fos.getBrowserStorageEffectForKey("recently-used-operators", {
```

Comment:
intentionally changing the persistence key here as it's not compatible prior values